### PR TITLE
Allocating unknown district data accurately

### PR DESCRIPTION
**Pull request by alphasingh**
Allocating the district data of unknown key to appropriate districts with latest data of the confirmed cases. Also, adding Patiala with 1 latest confirmed case in Punjab as of today, 29 March 2020.
Refer to [Latest Updates](https://en.wikipedia.org/wiki/2020_coronavirus_pandemic_in_Punjab,_India) for more details. Also updating the **lastupdatedtime** values as per the given source data.

### CODE REVIEW

**Comment by alphasingh** [comment]
As of now, a total of 7 districts have confirmed cases of patients in Punjab.

**Comment by alphasingh** [comment]
Changes for incorrect district names resolved, such as 
"SHAHID BHAGAT SINGH NAGAR" and "S.A.S. NAGAR"

File: state_district_wise.json [hunk 1]
```diff
@@ -466,32 +466,32 @@
...
+        "lastupdatedtime": "2020-03-09T10:00:00Z"
       },
-      "Shahid Bhagat Singh Nagar": {
+      "Shaheed Bhagat Singh Nagar": {
         "confirmed": 19,
-        "lastupdatedtime": ""
+        "lastupdatedtime": "2020-03-26T10:00:00Z"
       },
-      "S.A.S. Nagar": {
+      "Sahibzada Ajit Singh Nagar": {
```
Comment (alphasingh) [inline state_district_wise.json]: Refer to #31 as SAS Nagar would not reflect in the actual Map

File: state_district_wise.json [hunk 2]
```diff
@@ -466,32 +466,32 @@
   "Punjab": {
     "districtData": {
       "Amritsar": {
-        "confirmed": 2,
-        "lastupdatedtime": ""
+        "confirmed": 1,
+        "lastupdatedtime": "2020-03-09T10:00:00Z"
       },
-      "Shahid Bhagat Singh Nagar": {
+      "Shaheed Bhagat Singh Nagar": {
```
Comment (alphasingh) [inline state_district_wise.json]: Refer to #31 for actual mapping district names in Map

**Comment by JunaidBabu** [comment]
@alphasingh thank you for raising this issue. These files are auto-updated every 5 minutes from the Google sheets. I will report this to the sheet maintainers.

@sudevschiz please take care of this

**Pull request closed without merging**